src/content/api/cli.md

```diff
@@ -84,6 +84,28 @@ Name of template to generate.
 
 To generate a project without questions. When enabled, default answer for each question will be used.
 
+### Info
+
+Outputs information about your system.
+
+```bash
+npx webpack info [options]
+```
+
+**example**
+
+```bash
+npx webpack info --output=json
+```
+
+**Options**
+
+**`--output`**
+
+`string`
+
+To get the output in specified format (accept `json` or `markdown`).
```

Comment:
https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types

```suggestion
`string : 'json' | 'markdown'`

To get the output in specified format.
```